frontend/src/store/modules/shoots/getters.js

```diff
@@ -131,6 +139,18 @@ export function getSortVal (rootGetters, item, sortBy) {
 
 export default {
   filteredItems (state) {
+    if (state.freezeSorting) {
+      // When state is freezed, do not include new items
+      return compact(map(state.freezedShootSkeletons, skeletonItem => {
+        const storeItem = find(state.filteredShoots, item => {
+          return item.metadata.uid === skeletonItem.metadata.uid
+        })
```

Comment:
```suggestion
        const storeItem = find(state.filteredShoots, ['metadata.uid', skeletonItem.metadata.uid])
```

Reply:
Yes you are right we can simplify this for all the mentioned find functions. This changed a lot over time thus I did not yet simplify it. Will do so now...